src/slash-bedrock/libexec/brl-which

```diff
@@ -67,6 +68,19 @@ which_bin() {
 	done
 }
 
+which_bin_all() (
+	min_args "${#}" "1"
```

Comment:
I didn't test this before pushing, and it appears `min_args` doesn't work reliably from inside functions. Currently, this happens, and it's not immediately obvious to me how to fix it:

```
$ brl which -ab
ERROR: Insufficient arguments, see `--help`.
ERROR: Unexpected error occurred.
````